content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md

```diff
@@ -0,0 +1,118 @@
+- Automatically mount /dev/kmsg for log enricher usage if running with CRI-O and an allowed `io.kubernetes.cri-o.Devices` annotation. (#479, @saschagrunert)
+- Deploying kube-rbac-proxy sidecar in SPOD for exposing metrics via the new `metrics-spod` and `metrics-controller-runtime` services. (#424, @saschagrunert)
+- SPO's ProfileRecording CRD ProfileRecording which allows the admin to
+  record workloads and create security policies was extended to allow
+  recording SELinux profiles as well. In order to record a SELinux profile
+  for a workload, set ProfileRecording.Spec.Kind to SelinuxProfile. (#592, @jhrozek)
+- Switched to unix domain sockets for the GRPC servers. (#631, @saschagrunert)
+- This patch re-adds the no_bpf build tag triggered by the BPF_ENABLED=0 tag
+  environment variable if set to 0. A developer can then build SPO without the
+  built-in BPF support by running:
+  BPF_ENABLED=0 make
+  This is useful to build SPO in environments with older dependencies
+  that don't allow building the in-tree BPF-based recorder. (#690, @jhrozek)
+- Update example base profiles to their recent runtime versions. (#543, @saschagrunert)
+- `spod` can load and unload AppArmor profiles into clusters host servers.
+  `spod` now runs as `root` and `privileged` when apparmor is enabled. (#680, @pjbgf)
+
+### Documentation
+
+- Added documentation about how to record profiles by using the log enricher. (#521, @saschagrunert)
+- Added documentation how to use the automatically deployed `ServiceMonitor` with OpenShift as example platform. (#460, @saschagrunert)
+- Added log enricher documentation to installation-usage.md. (#498, @saschagrunert)
+- Added metrics documentation to `installation-usage.md`. (#449, @saschagrunert)
+- Added table of contents to installation documentation. (#493, @saschagrunert)
+- Changed documentation to reference `main` instead of `master` as default git branch. (#706, @saschagrunert)
+- Fixed header links containing source code in `installation-usage.md` (#606, @saschagrunert)
+
+### Bug or Regression
+
+- Do not retry container ID retrieval on container creation failures any more. (#612, @saschagrunert)
+
+### Other (Cleanup or Flake)
+
+- An OpenShift deployment manifest was included in deploy/openshift.yaml (#695, @JAORMX)
+- Bumps
+  golang.org/x/text to fix advisory GO-2021-0113 (#655, @pjbgf)
+- Log enricher now requires running auditd (`/var/log/audit/audit.log`) (#487, @saschagrunert)
+- Log libseccomp version on operator startup. (#556, @saschagrunert)
+- Removed CPU limits from SPOD and added resource requests/limits to manager and webhook. (#550, @saschagrunert)
+- The directory /etc/selinux.d used to be mounted on the hosts in previous SPO versions.
+  This is no longer the case, the directory was converted to an emptyDir instead,
+  reducing the number of required host mounts. (#698, @jhrozek)
+- The securityprofilenodestatus CR now links with the security profile its status
```

Comment:
oops, I ended up changing the patch to use sha256 but the release note was not updated..can we change the text to say sha256?

Reply:
You can just change it in the PR, and I can re-generate the notes later on.